src/correct-round.rkt

```diff
@@ -442,18 +443,18 @@
      (define x-exp (ival-max-log2-approx x))
      (define out-exp (ival-min-log2-approx output))
      
-     (list (max 0 (+ 1 x-exp) (+ 2 (- x-exp out-exp))))]
+     (list (+ 1 x-exp) (+ 2 (- x-exp out-exp)))]
     
     [(ival-atan2)
-     ; log[Гatan2]'x = log[Гatan2]'y = log[xy / ((x^2+y^2)*atan2)] <= log[x] + log[y] - 2*max[logx, logy] - log[atan2]
+     ; log[Гatan2]'x = log[Гatan2]'y = log[xy / ((x^2+y^2)*atan2)] <= log[x] + log[y] - 2*min[logx, logy] - log[atan2] + 1
      (define x (first srcs))
      (define y (second srcs))
      
      (define x-exp (ival-max-log2-approx x))
      (define y-exp (ival-max-log2-approx y))
      (define out-exp (ival-min-log2-approx output))
      
-     (make-list 2 (max 0 (- (+ x-exp y-exp) (* 2 (max x-exp y-exp)) out-exp)))]
+     (make-list 2 (+ (- (+ x-exp y-exp) (* 2 (min x-exp y-exp)) out-exp) 1))]
```

Comment:
This one was tricky. Turned out that I was over approximating sum of logs, when it actually had to be under approximated since we subtract this value. The overall exponent will be overapproximated with this change. Previously it was not enough